docker/docker_image_src.go

```diff
@@ -65,11 +73,37 @@ func (s *dockerImageSource) GetManifest() (manifest types.ImageManifest, unverif
 	if res.StatusCode != http.StatusOK {
 		return nil, "", errFetchManifest{res.StatusCode, manblob}
 	}
+	// TODO(remove, already set in manifest, below): Miloslav?
```

Comment:
Well, the commit keeps the variable but removes its only user :) Do either, but not both. Or perhaps reconsider why `skopeo inspect` even wants to print that value, I don’t know the context.